lib/gitea.ex

```diff
@@ -217,7 +217,7 @@ defmodule Gitea do
   def commit(org_name, repo_name, params) do
     repo = local_git_repo(org_name, repo_name)
     # Add all files in the repo
-    {:ok, _output} = Git.add(repo, ["."])
+    {:ok, _output} = Git.add(repo, [".", "-f"])
```

Comment:
```sh
git help add
```
> -f, --force: Allow adding otherwise ignored files.

I've tested this by creating git repository with a file in .gitignore then using the `git add . -f` and it does indeed add it to "changes to be committed".

Do we want this to happen?

Reply:
Sorry I'm not sure I understand what you mean.
I don't see the link between creating the git submodule for the `test-repo` folder and having the `Gitea.commit` function using `git add . -f` to add all the files before committing.
Reading the tests code to see if I miss something

Reply:
The reason was that I was getting an error message on my `localhost`. 🙄 
I've removed the `-f` flag. ✂️ 
If the tests pass on `CI` I will just figure it out on `localhost`. 👍